### ISSUE TITLE
Alt segment_info table

### ISSUE THREAD

**Issue by pwoods25443** [GlobalFishingWatch/pipe-segment#64]
Temporary workaround to deal with the orphan vessel_id problem.

Build a table `alt_segment_identity` as a daily table based on `position_messages` like this

```sql
SELECT
  seg_id,
  vessel_id,
  ssvid
FROM
  `world-fishing-827.pipe_production_b.position_messages_20180820`
GROUP BY
  seg_id,
  vessel_id,
  ssvid
```

Then consolidate that table into `alt_segment_info` with 

```sql
SELECT
  seg_id,
  vessel_id,
  ssvid
FROM
  `world-fishing-827.pipe_production_b.alt_segment_identity_*`
GROUP BY
  seg_id,
  vessel_id,
  ssvid
```

**Comment by pwoods25443** [comment]
Might not need this if #66 works

**Comment by pwoods25443** [comment]
Closed in favor of #66
